### PR TITLE
updated landing and partial for inner

### PR DESCRIPTION
**Pull request by webuimaker**
Updated landing page and inner page partial

### CODE REVIEW

**Comment by simar7** [comment]
@webuimaker Is this ready for review? If it's not, please mark it as a "Draft" until then. https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by webuimaker** [comment]
> @webuimaker Is this ready for review? If it's not, please mark it as a "Draft" until then. https://github.blog/2019-02-14-introducing-draft-pull-requests/

Landing page done and inner page in progress but I will keep this is in next time

**Comment by webuimaker** [comment]
Please merge and check I have update inner page too.

File: .vscode/configurationCache.log [hunk 1]
```diff
@@ -0,0 +1 @@
+{"buildTargets":[],"launchTargets":[],"customConfigurationProvider":{"workspaceBrowse":{"browsePath":[]},"fileIndex":[]}}
```
Comment (simar7) [inline .vscode/configurationCache.log:1]: Remove

File: .vscode/dryrun.log [hunk 2]
```diff
@@ -0,0 +1,4 @@
+make.exe --dry-run --always-make --keep-going --print-directory
+'"make.exe"' is not recognized as an internal or external command,
+operable program or batch file.
+
```
Comment (simar7) [inline .vscode/dryrun.log:4]: Remove

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
+}
```
Comment (simar7) [inline .vscode/settings.json:3]: Remove

File: content/cspm/_index.md [hunk 4]
```diff
@@ -15,3 +16,26 @@ avd_page_type: cloudsploit_page
...
+- .NET Framework Version
+- Authentication Enabled
+- Client Certificates Enabled
+- HTTP 2.0 Enabled
+- HTTPS Only Enabled
+- Identity Enabled
+- Java Version
+- PHP Version
+- Python Version
+- TLS Version Check
```
Comment (simar7) [inline content/cspm/_index.md:41]: What are these?

File: content/cspm/aws/acm/acm-certificate-validation.md [hunk 5]
```diff
@@ -24,11 +24,31 @@ breadcrumb_remediation_child_name: ACM
...
+bla bla bla bla bla bla bla bla bla
+bla bla bla bla bla bla bla bla bla
+bla bla bla bla bla bla bla bla bla
+bla bla bla bla bla bla bla bla
+{{< /codetab >}}
+
+
+{{< /tabs >}}
+
+
```
Comment (simar7) [inline content/cspm/aws/acm/acm-certificate-validation.md:54]: What's this for?

File: themes/aquablank/layouts/partials/featured.html [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+							<a href="#" class="button is-small">Alpine</a>
+						</div>
+				</div>
+			</div>
+
+		</div><!-- columns -->
+
+	</div><!-- container -->
+
+</div><!-- projects_wrap -->
```
Comment (simar7) [inline themes/aquablank/layouts/partials/featured.html:61]: These are hardcoded, how do we plan to generate this automatically?

File: themes/aquablank/layouts/partials/home.html [hunk 7]
```diff
@@ -5,16 +5,25 @@
...
 							<div class="content search_results_list_wrap"><ul id="search_results_list"></ul></div>
 						</div>
 					</div>
+
+					<h3 class="fadeInUp animationDelay_2 my-2">Or Browse by Category</h3>					
+						<ul class="categories-list">
+							{{range .Sections}}
+								<li class="is-inline-block"><a href="{{.Permalink}}">{{.Title}}</a> </li>
+							{{end}}
+
```
Comment (simar7) [inline themes/aquablank/layouts/partials/home.html:24]: The links don't work here. For example /appshield returns back to the home page.

File: themes/aquablank/layouts/partials/list_cloudsploit.html [hunk 8]
```diff
@@ -35,6 +36,28 @@ <h1 class="title page_title is-spaced fadeInUp animationDelay_1">Remediation Gui
...
+							<li class=""><a href="{{.Permalink}}">GCP</a> </li>
+							<li class=""><a href="{{.Permalink}}">Docker</a> </li>
+							<li class=""><a href="{{.Permalink}}">Kubernetes</a> </li>
+							<li class=""><a href="{{.Permalink}}">Runtime Threats</a> </li>
+							<li class=""><a href="{{.Permalink}}">CVEs</a> </li>
+						</ul>
+					
+                    </div>
+                   
+                </div>
```
Comment (simar7) [inline themes/aquablank/layouts/partials/list_cloudsploit.html:53]: Also don't work. They are all pointing to /cspm. What were you trying to implement here?

File: themes/aquablank/layouts/partials/startnow.html [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+                        <h2 class="is-size-4 has-text-weight-medium my-2">start protecting your cloud</h2>
+                        <p class="my-2">
+                            <a href="#" target="_blank" class="button is-primary are-large is-large is-fullwidth">Start Now</a>
+					
+                        </p>
+
+                       
+                    </div>
+                   
+                </div>
```
Comment (simar7) [inline themes/aquablank/layouts/partials/startnow.html:13]: Is this the right size? Looks quite off for me: 
image

File: content/cspm/aws/acm/acm-certificate-validation.md [hunk 10]
```diff
@@ -12,7 +12,7 @@ breadcrumb_remediation_child_name: ACM
 ---
 ### Quick Info
 
-| | |
+| Issue ID | AZU001 |
```
Comment (simar7) [inline content/cspm/aws/acm/acm-certificate-validation.md:15]: There isn't any ID associated here. Were you trying to use this as a key for something? If so use The actual Plugin Title has it is unique.

File: content/cspm/aws/acm/acm-certificate-validation.md [hunk 11]
```diff
@@ -24,11 +24,31 @@ breadcrumb_remediation_child_name: ACM
 
 ## Detailed Remediation Steps
 
+{{< tabs  Runtime IaC >}}
+
+{{< codetab >}}
```
Comment (simar7) [inline content/cspm/aws/acm/acm-certificate-validation.md:29]: What happens if a policy doesn't have both steps? Do we still show the tabs?

File: themes/aquablank/layouts/partials/page_cloudsploit.html [hunk 12]
```diff
@@ -63,6 +70,57 @@ <h1 class="title page_title is-spaced fadeInUp animationDelay_1">{{ .Params.disp
             </div><!-- column is-8 -->
 
             <div class="column is-4-desktop is-6-tablet is-12-mobile fadeInUp animationDelay_2">
+                <div class="round-aera">
+                    <div class="box is-rounded has-text-danger	 is-size-3">
+                        High
+                    </div>
```
Comment (simar7) [inline themes/aquablank/layouts/partials/page_cloudsploit.html:76]: We need to pull this from the page not hardcode as different things will have different severity classification.

**Review by simar7 — CHANGES_REQUESTED** [review]
Left some comments. For the UI specific pages, could you also attach a screenshot to show the changes with the PR comments going forward?

**Pull request closed without merging**